main.tf

```diff
@@ -14,12 +14,26 @@ locals {
   domain_name = length(module.dns_and_certificates) == 0 ? "" : module.dns_and_certificates[0].domain_name
 }
 
+
+locals {
+  create_vpc = var.use_an_existing_vpc == false ? true : false
```

Comment:
Please simplify. Actually "create_vpc" is not needed to be created.

Reply:
done